spec/oas_parser/endpoint_spec.rb

```diff
@@ -120,6 +120,36 @@
     end
   end
 
+  describe '#basic_auth?' do
+    it 'returns a boolean indicating if the endpoint uses Basic Auth authentication' do
+      expect(@endpoint.basic_auth?).to eq(false)
+    end
+
+    context 'when the definition provides a security scheme with JWT' do
```

Comment:
Should this be `basic` rather than `JWT`?

Reply:
Good spot... thanks.

Reply:
👍 Good spot, thanks!